common/src/main/java/com/tc/async/impl/StageManagerImpl.java

```diff
@@ -47,7 +47,8 @@
  * @author steve
  */
 public class StageManagerImpl implements StageManager {
-
+  private static final int         DEFAULT_MAX_STAGE = TCPropertiesImpl.getProperties()
```

Comment:
If I understand correctly, this signifies capacity of sink of each SEDA stage i.e.  maximum size of queue. So why not naming it like `DEFAULT_STAGE_QUEUE_MAX_SIZE` or `DEFAULT_STAGE_SINK_CAPACITY`?